packages/quantic/force-app/main/default/lwc/quanticViewedByCustomerBadge/quanticViewedByCustomerBadge.html

```diff
@@ -0,0 +1,17 @@
+<template>
+  <template if:true={hasInitializationError}>
+    <c-quantic-component-error component-name={template.host.localName}>
+    </c-quantic-component-error>
+  </template>
+  <template if:true={shouldDisplayBadge}>
+    <div class="slds-align_absolute-center">
+      <lightning-icon
+        icon-name="utility:profile_alt"
+        class="slds-var-p-right_xx-small"
+        alternative-text={labels.viewedByCustomer}
+        size="xx-small"
+      ></lightning-icon>
+      <div class="viewed-by-customer__label">{labels.viewedByCustomer}</div>
+    </div>
+  </template>
```

Comment:
If there is no result it's most likely an error from the developer setting up the Quantic interface, and that's why I'm not keeping it silent. 

Basically this could happen when the developer using Quantic puts in his result template the following: 
```
<c-quantic-viewed-by-customer></c-viewed-by-customer-badge>
```
Notice that the property `result` was not passed to the component.
In that case I think it would be very useful for the developer to fix his error even if it shows a red text on every result.

Other than a developer making an error while creating a result template, I don't see how this error would occur and that's why I think it should be visible.

Additionally we are having the same logic in other components like the QuanticResultDate, QuanticResultNumber...

Reply:
Okay if it's already there elsewhere.

It just becomes something that starts to feel "required" for every components